stable/fairwinds-insights/values.yaml

```diff
@@ -218,6 +218,11 @@ cronjobs:
     command: cluster_deletion
     schedule: "0 * * * *"
 
+  # -- Options for the refresh_jira_webhooks cronjob
+  refresh-jira-webhooks:
+    command: refresh_jira_webhooks
+    schedule: "0 0 25 * *"
```

Comment:
Wanna run this once a month at day  25?

Reply:
Yeah, changed to run at day 1 and day 15 of each month, there is no option to "after 25d"